twython/endpoints.py

```diff
@@ -199,28 +199,33 @@ def upload_video(self, media, media_type, size=None):
             'command': 'FINALIZE',
             'media_id': media_id
         }
-        response_finalize = self.post(upload_url, params=params)
 
-        # Stage 4: STATUS call if still processing
-        params = {
-            'command': 'STATUS',
-            'media_id': media_id
-        }
-        
-        processing_state = response_finalize['processing_info'].get('state', None)
+        # Only get the status if explicity asked to 
+        # Default to False
+        if check_progress == True:
```

Comment:
No worries! @michaelhelmick totally happy to ensure everything is compliant. appreciate the feedback!